packages/e2e/detox/integration/common/shared.ts

```diff
@@ -4,7 +4,6 @@ import { by, element, expect } from 'detox';
 Given("I'm running the example {string}", async (name: string) => {
   await device.launchApp({
     newInstance: true,
-    delete: true,
```

Comment:
Instead of deleting the app after every scenario and re-installing it, which takes time, modified the tests to leave a clean state (i.e. sign out after signing in)